src/module10/Main101.java

```diff
@@ -0,0 +1,19 @@
+package module10;
+
+public class Main101 {
+
+    public static void main(String[] args) throws Exception{
```

Comment:
Create a class with a main( ) that throws an object of class Exception inside a try block. Give the constructor for Exception a String argument. Catch the exception inside a catch clause and print the String argument. Add a finally clause and print a message to prove you were there.